episodes/13-narrative.md

```diff
@@ -4,70 +4,128 @@ teaching: 15
+you are teaching.
+This may mean you will need to subset your dataset or remove variables
+observed.
+You may want to also include and review a data dictionary in your lesson,
+explicitly taking the time to review the information included in the
+dataset.
+An additional factor to consider when choosing a dataset to include is 
+the license.
+You want to find a dataset where the data provider allows for you to freely use it.
+The best option is a dataset with a CC0 (Public Domain Dedication) license, as
+other licenses may have more ambiguity around data reuse.
+Lessons included in the [Carpentries Incubator][carpentries-incubator] will need to use CC0 licensed data.
+Even with a CC0 license, you will still want to follow best practice in 
+giving attribution to the data provider or collecting agency.
+
+:::::::::::::::: callout
+
+## More CC0 License Reading
+
+If you'd like to read more about CC0 and CC-BY, Katie Fortney wrote an excellent 
+[blogpost on why CC-BY is not always a good fit](https://osc.universityofcalifornia.edu/2016/09/cc-by-and-data-not-always-a-good-fit/).
+
+::::::::::::::::::
+
+
+An example dataset used in the Data Carpentry Ecology lessons is the [Portal
+Project Teaching Database](https://figshare.com/articles/dataset/Portal_Project_Teaching_Database/1314459).  This dataset is an actual ecological research project that was simplified for teaching.
+The reuse of this dataset throughout the Data Carpentry Ecology lessons helps
+stitch together the process of data analysis on throughout the workshop, from
+data entry and cleaning to analysis and visualization.
+
+When deciding on a dataset, it is also important to consider the ethical use of
+each dataset considered.  *Does the data contain personally identifiable information?*
+*Was the data collected without permission from the groups or individuals included?*
+*Will the data be upsetting to learners in the workshop?*
+If the answer to any of these questions might be yes, you will need to do more
+research on it and continue to look for other options.  Commonly data is misused
+from historically excluded and exploited groups.
+The [CARE Principles for Indigenous Data Governance](http://doi.org/10.5334/dsj-2020-043) (Collective Benefit, Authority to Control, Responsibility, and Ethics) are good starting point
+for thinking about data sovereignty and considering the ethics of data collected
+about individual or groups of people.
+
+
+### Examples of Public Repositories
+
+When looking for data to reuse, consider public repositories in the subject area
+for your lesson or general data repositories such as:
+  - [Dryad](https://datadryad.org/)
+  - [The Data Cuartion Network's datasets](https://datacurationnetwork.org/datasets/)
+  - [The Offical Portal for European Data](https://data.europa.eu/)
+  - [DataONE](https://www.dataone.org/)
+- [The Official Portal for Argentina Data](https://www.datos.gob.ar/) - (In Spanish)
```

Comment:
```suggestion
  - [The Offical Portal for European Data](https://data.europa.eu/)
  - [The Official Portal for Argentina Data](https://www.datos.gob.ar/) - (In Spanish)
  - [DataONE](https://www.dataone.org/)
  
You may also want to look for ready to look for packages or built-in data from the tool you are teaching.
For example, [Palmer Penguins](https://cran.r-project.org/web/packages/palmerpenguins) is commonly used when teaching R.
[Tidy Tuesday](https://github.com/rfordatascience/tidytuesday) or [Datos de Miércoles](https://github.com/cienciadedatos/datos-de-miercoles) are 
also good places to look for new datasets that meet your criteria.
```

Reply:
@yabellini Does this addition address your suggestion?  Also is the link and capitalization for Datos de Miércoles correct?

Reply:
Yes and yes, thank you!